src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,52 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+	@Test
+	@DisplayName("구분자로 문자열 분리")
+	void split() {
+		String data = "1,2";
+		String[] result = data.split(",");
+		assertThat(result).contains("1", "2");
```

Comment:
```suggestion
		String data = "1,2";
		String[] result = data.split(",");

		assertThat(result).contains("1", "2");
```

보통 assertion 전에 빈 줄을 넣어서, 무엇을 검증하는지 눈에 잘 들어오도록 만들곤 해요.